spec/models/queries/statistics_query_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Queries::StatisticsQuery do
+  describe "#call" do
+    it "calls the defined association" do
```

Comment:
In this test, there is no explicit verification phase. There is a setup phase and execution phase but no verification. A better way you can go is to use test spies like this:
```ruby
 allow(subject).to receive(:questions_asked)
 allow(subject).to receive(:answers_given)
 allow(subject).to receive(:join_associations)

subject.call

 expect(subject).to have_received(:questions_asked)
 expect(subject).to have_received(:answers_given)
 expect(subject).to have_received(:join_associations)
```
Note that you don't need to use `described_class.new` in this case. `subject` is `described_class.new`